src/curve_fitting/core.clj

```diff
@@ -23,6 +23,14 @@
     (quil/curve-vertex x y))
   (quil/end-shape))
 
+(defn draw-point-borders
+  [pixel-x pixel-y]
+  "Draws a white border around a point to make it easier to distinguish between
+  points and lines."
+  (quil/fill 255 255 255 255)
+  (quil/ellipse
+    pixel-x pixel-y (+ 2 point-pixel-radius) (+ 2 point-pixel-radius)))
```

Comment:
Probably time for us to introduce a `curve-fitting.util.quil` namespace with `circle` in it.

Reply:
I'll make an issue for that.